src/test/resources/examples/enumExamples/models/Models.kt

```diff
@@ -2,10 +2,17 @@ package examples.enumExamples.models
 
 import com.fasterxml.jackson.annotation.JsonProperty
 import com.fasterxml.jackson.annotation.JsonValue
+import javax.validation.Valid
 import kotlin.String
 import kotlin.collections.List
 import kotlin.collections.Map
 
+data class BarBar(
```

Comment:
I don't see an easy way to not generate these. I think we may want them sometimes